ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-4.19-nightly-multi.yaml

```diff
@@ -69,7 +69,28 @@ tests:
     test:
     - chain: openshift-qe-node-density-heavy
     workflow: baremetal-lab-upi
-  timeout: 6h0m0s
+  timeout: 7h0m0s
+- always_run: false
+  as: node-density-heavy-custom-cidr-3nodes
+  capabilities:
+  - intranet
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKERS: "2"
+      ADDITIONAL_WORKERS_DAY2: "false"
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      architecture: arm64
+      masters: "3"
+      workers: "1"
```

Comment:
Customize the CLUSTER_NETWORK_CIDR and CLUSTER_NETWORK_HOST_PREFIX here as envs.